src/components/Guilds/ProposalStatus/index.tsx

```diff
@@ -84,17 +85,23 @@ const ProposalStatus: React.FC<ProposalStatusProps> = ({
 
   const statusDetail = useMemo(() => {
     if (!proposal?.endTime) return null;
-
-    if (proposal.state === ProposalState.Submitted) {
-      const currentTime = moment();
-      if (currentTime.isSameOrAfter(proposal.endTime)) {
-        return 'Ended';
-      } else {
-        return 'Active';
-      }
+    switch (proposal.state) {

```

Comment:
Had a chat about this with @Kenny-Gin1. We need to create a separate issue to parse these proposal states properly.

Reply:
Nice, going to make a few tasks soon for detailed specs we need for things like proposal state as well as a few other things to ensure we have one correct util for displaying this